msteams-platform/toolkit/Microsoft-365-Agents-Toolkit-CLI.md

```diff
@@ -0,0 +1,517 @@
+| `--env`                 | No       | Specifies the environment name for the project scaffolded by Microsoft 365 Agents Toolkit.                          |
+| `--env-file`            | No       | Specifies the .env file that defines the variables to replace in the app manifest template file.                    |
+| `--manifest-file`       | No       | Specifies the app manifest file path. The default value is `./appPackage/manifest.json`.                            |
+| `--package-file`        | No       | Specifies the zipped app package file path.                                                                         |
+| `--output-folder`       | No       | Specifies the output folder containing the manifest(s). The default value is `./appPackage/build`.                  |
+| `--output-package-file` | No       | Specifies the output zipped app package file path. The default value is `./appPackage/build/appPackage.${env}.zip`. |
+| `--folder -f`           | No       | Specifies the project's root folder. The default value is `./`.                                                     |
+
+## `atk preview`
+
+The `atk preview` command allows you to preview your app during development. This command provides a real-time view of your app, helping you identify and fix issues before deployment.
+
+### `atk preview` parameters
+
+The following table lists the parameters available for `atk preview`:
+
+| Parameter              | Required | Description                                                                                                                                                                                                                                              |
+| :--------------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--m365-host -m`       | No       | Preview the application in Teams, Outlook, or the Microsoft 365 app. The available options are `teams`, `outlook`, and `office`. The default value is `teams`.                                                                                           |
+| `--manifest-file`      | No       | Specifies the app manifest file path. The default value is `./appPackage/manifest.json`.                                                                                                                                                                 |
+| `--run-command -c`     | No       | The command starts the service and works only in local environment. If undefined, it auto-detects the appropriate command from project type (`npm run dev:teamsapp`, `dotnet run`, or `func start`). If empty, it skips starting the local service.      |
```

Comment:
Is this correct commant?
dev:teamsapp or it should be dev:atk?

Reply:
This one is NOT updated since the very beginning, so now it still is "npm run dev:teamsfx". This command is usually called by templates in the toolkit, not executed by users. Update to teamsfx